src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ReviewController.kt

```diff
@@ -75,6 +79,22 @@ class ReviewController(
     )
   }
 
+  @PutMapping("/schedule-status")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  @PreAuthorize(HAS_EDIT_REVIEWS)
+  @Transactional
+  fun updateLatestReviewScheduleStatus(
+    @Valid
+    @RequestBody createReviewScheduleStatusRequest: CreateReviewScheduleStatusRequest,
+    @PathVariable @Pattern(regexp = PRISON_NUMBER_FORMAT) prisonNumber: String,
+  ) {
+    reviewScheduleService.updateLatestReviewScheduleStatus(
+      prisonNumber,
+      createReviewScheduleStatusRequest.prisonId,
+      createReviewScheduleStatusRequest.status.value,
```

Comment:
Re: my comment on the service class, if we can change the method arg from a sting to the enum type, then you'll need a method here, much like the `toSentenceType` one below.

I know its a few more lines of code, but one of the things I like about kotlin are things like the exhaustive when which means we can be much more type safe in places like this, and we'll get a compilation error if we add a new value in the domain enum but forget to do it in the API (or visa versa)

What do you you think?